lib/service/service.go

```diff
@@ -3206,6 +3233,53 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 			log.Info("Exited.")
 			return nil
 		})
+
+		if listeners.minimalReverseTunnelWebEnabled {
+			minimalListener, err := multiplexer.NewWebListener(multiplexer.WebListenerConfig{
```

Comment:
Could we move the contents of this block to a named func? `initMinimalReverselTunnel()` or something else you prefer?